packages/ckeditor5-dev-env/lib/release-tools/utils/transformcommitforckeditor5devpackage.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const transformCommitForCKEditor5Package = require( './transformcommitforckeditor5package' );
+const getChangedFilesForCommit = require( './getchangedfilesforcommit' );
+
+/**
+ * Parses a single commit for `ckeditor5-dev` package.
+ *
+ * @param {Commit} commit
+ * @param {Object} {context}
+ * @param {Boolean} context.displayLogs Whether to display the logs.
+ * @param {Object} context.packageData Content from the 'package.json' for given package.
+ * @returns {Commit}
+ */
+module.exports = function transformCommitForCKEditor5DevPackage( commit, context ) {
```

Comment:
We accept the commit only if its files are located in current processing package. [See the tests.](https://github.com/ckeditor/ckeditor5-dev/pull/154/files#diff-c4d9754e1f61e34742144bb2dcc0d345R62)